src/common/res/features/goal-indicator/main.js

```diff
@@ -0,0 +1,106 @@
+(function poll() { 
+  // Waits until an external function gives us the all clear that we can run (at /shared/main.js)
+  if ( typeof ynabToolKit !== "undefined"  && ynabToolKit.pageReady === true ) {
+  
+    ynabToolKit.featureOptions.goalIndicator = true;
+    ynabToolKit.goalIndicator = function ()  { // Keep feature functions contained within this
+    	var entityManager = ynab.YNABSharedLib.getBudgetViewModel_AllAccountTransactionsViewModel()._result.getEntityManager();
```

Comment:
I would suggest using ynab.YNABSharedLib.defaultInstance.entityManager

If YNAB does some performance optimizations where the All Accounts view is not loaded until it is opened, this will break.